.travis.yml

```diff
@@ -32,4 +33,4 @@ deploy:
   provider: script
   script: heroku container:release --app jol-todoapp web
   on:
-    branch: master
+    branch: master
```

Comment:
You should be able to allow your m9 branch to deploy (on the branch, but not PR build) by just adding the relevant branch here:
```
  on:
    branch:
      - master
      - module9
``